ODIN_II/SRC/netlist_create_from_ast.cpp

```
@@ -510,6 +514,10 @@ signal_list_t* netlist_expand_ast_of_module(ast_node_t** node_ref, char* instanc
                 break;
             }
             case ALWAYS:
+                if (chk == true) {
+                    error_message(NETLIST, chk_loc, "%s",
+                              "Output cannot be defined as a wire\n");
```

Please throw an informative error message, since output usually should be a wire except in a specific case like this which should be a data type